plonecli/cli.py

```diff
@@ -81,6 +81,8 @@ def create(context, template, name):
         raise NoSuchValue(
             context.command.name, template, possibilities=reg.get_templates()
         )
+    if set(name).intersection(set(string.punctuation)):
```

Comment:
As i mentioned, a dash or a underscore is a legal sign for a Python Project name.
What we need to do is, to normalizes it for the module names and in ZCML.
For now you can add a note to the docs, as you suggested.